test/test.py

```diff
@@ -83,5 +93,9 @@ def test_check_user_login(self, instance):
       timeout = 10
       socket.setdefaulttimeout(timeout)
       url = instance.returnValues['Cloudera.manager']
-      response = requests.get(url, auth=requests.auth.HTTPBasicAuth('admin', 'admin'))
-      self.assertEqual(200, response.status_code)
+      
+      @eventually(AssertionError)
+      def eventually_assert():
+        response = requests.get(url, auth=requests.auth.HTTPBasicAuth('admin', 'admin'))
```

Comment:
These creds are defaults from manifest. Ideally they should be part of tests configuration of course.